docs/UserGuide.adoc

```diff
@@ -261,11 +261,109 @@ Address book data are saved in the hard disk automatically after any command tha
 There is no need to save manually.
 
 // tag::dataencryption[]
-=== Encrypting data files `[Coming in v2.0]`
+=== Encrypting data files : `encrypt` `[Coming in v2.0]`
 
-_{explain how the user can enable/disable data encryption}_
+Encrypt the specified person from the address book so that password will be required to view the information of the specified personz. +
+Format: `encrypt INDEX`
+
+****
+* Encrypts the person at the specified `INDEX`.
+* The index refers to the index number shown in the most recent listing.
+* The index *must be a positive integer* 1, 2, 3, ...
+****
+
+Examples:
+
+* `list` +
+`encrypt 3` +
+Encrypts the 3rd person in the address book.
+* `find Betsy` +
+`encrypt 1` +
+Encrypts the 1st person in the results of the `find` command.
 // end::dataencryption[]
 
+=== Login : `login` `[Coming in v2.0]`
+
+Allow user to login to the social media specified. +
+Format: `loging SOCIALMEDIA`
```

Comment:
typo: loging -> login
Though I'm coding with a different format, you can merge this and I'll modify the description later.